### ISSUE TITLE
ACC-264.1, ACC-271.1 (Level A) _ Modify Modules _ Navigation menu _ Additional tab stop with speech output

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3151]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	List item
Affected Check Point:	ACC-264.1, ACC-271.1
Short text:	ACC-264.1, ACC-271.1 (Level A) _ Modify Modules _ Navigation menu _ Additional tab stop with speech output 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Now navigate to any  menu item from the left side navigation and press tab key and observe the focus.

3. Issue Type: Manual
Observed Behavior:
On accessing any  menu item from the left side navigation and pressing tab key, there is an extra tab stop with speech output ‘First Column region Close drawer button’ whereas visually there is no such button in the page.
Screenshot:
 
Case 2:
 

Expected Behavior:
On accessing any  menu item from the left side navigation and pressing tab key, the focus should move the first interactive element in the main region i.e. ‘Information’ button and additional speech output of a not visible button must be removed.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1, ACC-271.1 (Level A) _ Modify Modules _ Navigation menu _ Additional tab stop with speech output.docx](https://github.com/user-attachments/files/16660160/ACC-264.1.ACC-271.1.Level.A._.Modify.Modules._.Navigation.menu._.Additional.tab.stop.with.speech.output.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660161/Issue.processing-09.2021.002.docx)
